### PR TITLE
[REF] MQT: Migration to python3

### PR DESCRIPTION
**Pull request by moylop260**
[REF] MQT: Migration to python3
  - [REF] remove relative import and reduce expected lint errors
  - [REF] py3: xmlrpclib compatilibity
  - [FIX] test_server: Write bytes for binary file and print string
  - [REF] Use br for regex

[REF] MQT: Compatibility with new api odoo

  - [REF] compatible with old and new api
  - [REF] remove deprecated _columns
  - [REF] .travis.yml: Remove 6.1 and 7.0 odoo test since that 11 version don't support _columns and 6.1 and 7.0 just suppport _columns

[REF] .travis.yml: Enable py3 environment

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -8,35 +8,54 @@ cache:
...
+      env:
+        VERSION="9.0" TESTS="1" LINT_CHECK="0"
+        VERSION="" LINT_CHECK="1" TESTS="0" PYLINT_EXPECTED_ERRORS="19" TRAVIS_PULL_REQUEST="false"  # To check VERSION empty or unset case.
+    - python: 2.7
+      env:
+        VERSION="9.0" TESTS="1" LINT_CHECK="0"
+        INCLUDE="broken_no_access_rule" SERVER_EXPECTED_ERRORS="1"  # test errors are detected
+    - python: 3.4
+      env:
+        VERSION="9.0" TESTS="1" LINT_CHECK="0"
```
Comment (pedrobaeza) [inline .travis.yml]: Version 9.0?

Comment (moylop260) [reply .travis.yml]: I'm not changing that part:
 - https://github.com/OCA/maintainer-quality-tools/blob/164f6801de47dfff42045a6c4216924621e97e1b/.travis.yml#L26



I just changed to use multi-python environment.

Comment (pedrobaeza) [reply .travis.yml]: Yeah, but 9.0 doesn't work with Python 3

Comment (moylop260) [reply .travis.yml]: Oh, I got it your point.
Thanks for clarify.

We had a global configuration using `VERSION=9.0` but I didn't find a way to use a `global` configuration using multi-python platform. (overwritten from particular one)
Then I just copied and paste in one line the global configuration in order to see the same global effect and overwrite in the next line the particular one.

What do you think about?

Comment (moylop260) [reply .travis.yml]: @pedrobaeza 
I just removed "global" environment variable and I'm using the required.

**Comment by lepistone** [comment]
@moylop260 are you sure travis doesn't support Python 3.5? From https://docs.travis-ci.com/user/languages/python/#Specifying-Python-versions it looks like they do. I'd change that to 3.5 which is the version supported by Odoo. Thanks!

**Comment by pedrobaeza** [comment]
Merging as this has been green on VX branch.

**Pull request merged**